packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -572,26 +613,29 @@ class ContextMenu extends MenuBase {
   }
 
   _getOverlayOptions(): OverlayProperties {
```

Comment:
Use extended types with private definition where it's possible from ts directory